docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,444 @@
+class GrandParentWithOutputInput extends Bundle {
+  val alignedCoerced = Output(new ParentWithOutputInput())
+  val flippedCoerced = Input(new ParentWithOutputInput())
+}
+class MyModule3 extends Module {
+  val g = Wire(new GrandParentWithOutputInput)
+}
+```
+
+Remember that `Output(gen)/Input(gen)` recursively strips the `Flipped` of any recursive children.
+This makes every member of `gen` aligned with every other member of `gen`.
+
+Consider the following alignments between grandparent and grandchildren. Because `alignedCoerced` and `flippedCoerced` are aligned with all their recursive members, they are fully aligned. Thus, only their alignment to `g` influences grandchildren alignment:
+ - `g` is aligned w.r.t `g.alignedCoerced.alignedChild`
+ - `g` is aligned w.r.t `g.alignedCoerced.flippedChild`
+ - `g` is flipped w.r.t `g.flippedCoerced.alignedChild`
+ - `g` is flipped w.r.t `g.flippedCoerced.flippedChild`
+
+Consider the following alignment relationships starting from `g.alignedCoerced` and `g.flippedCoerced`. *Note that whether `g.alignedCoerced` is aligned/flipped relative to `g` has no effect on the aligned/flipped relationship between `g.alignedCoerced` and `g.alignedCoerced.alignedChild` or `g.alignedCoerced.flippedChild` because alignment is only relative to the two members in question! However, because alignment is coerced, everything is aligned between `g.alignedCoerced`/`g.flippedAligned` and their children*:
+ - `g.alignedCoerced` is aligned w.r.t. `g.alignedCoerced.alignedChild`
+ - `g.flippedCoerced` is aligned w.r.t. `g.alignedCoerced.flippedChild`
+ - `g.alignedCoerced` is aligned w.r.t. `g.flippedCoerced.alignedChild`
+ - `g.flippedCoerced` is aligned w.r.t. `g.flippedCoerced.flippedChild`
+
+In summary, `Input(gen)` and `Output(gen)` recursively coerce children alignment, as well as dictate `gen`'s alignment to its parent bundle (if it exists).
+
+## Connecting components with fully aligned members
+
+For simple connections where all members of aligned (non-flipped) with one another, use `:=`:
+
+
+```scala mdoc:silent
+import chisel3._
+class FullyAlignedBundle extends Bundle {
+  val a = Bool()
+  val b = Bool()
+}
+class Example0 extends RawModule {
+  val incoming = IO(Flipped(new FullyAlignedBundle))
+  val outgoing = IO(new FullyAlignedBundle)
+  outgoing := incoming
+}
+```
+
+This generates the following Verilog, where each member of `incoming` drives every member of `outgoing`:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example0())
+```
+
+> You may be thinking "Wait, I'm confused! Isn't foo flipped and bar is aligned?" -- Noo! Whether foo is aligned with bar makes no sense; remember, you only evaluate alignment between members of the same component or Chisel type. Because components are always aligned to themselves, `bar` is aligned to `bar`, and `foo` is aligned to `foo`, there is no problem. Their relative flippedness to anything else is irrelevant.
```

Comment:
```suggestion
> You may be thinking "Wait, I'm confused! Isn't foo flipped and bar aligned?" -- Noo! Whether foo is aligned with bar makes no sense; remember, you only evaluate alignment between members of the same component or Chisel type. Because components are always aligned to themselves, `bar` is aligned to `bar`, and `foo` is aligned to `foo`, there is no problem. Their relative flippedness to anything else is irrelevant.
```